CHANGES.rst

```diff
@@ -1,6 +1,33 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-19)
+--------------------------
+
+- Improves command line tool documentation by grouping of commands and
+  command examples.
+- Centralises the way workflow name is passed to CLI commands.
+- Renames ``workflows`` to ``list`` and ``list`` to ``ls``.
+- Introduces ``du`` command for checking workspace disk usage.
+- Introduces ``mv`` command for moving files within workspace.
+- Introduces ``diff`` command for comparing two workflows.
+- Introduces ``rm`` command for removing files within workspace.
+- Introduces ``stop`` command for stopping a running workflow.
+- Introduces ``delete`` command for deleting a workflow.
+- Introduces ``run`` command which wraps ``create``, ``upload`` and ``start``
```

Comment:
What do you think about saying that `run` will, by parsing `reana.yaml`, automatically create a workflow, upload its assets/files and start its execution? The reference to the wrapping could still be added to the end between parenthesis, like: Introduces ``run`` command .... (wraps `create`, `upload` and `start` commands).